### PR TITLE
Redefine manufactured solution

### PR DESCRIPTION
**Pull request by jgonzalezusua**
**Description**
This PR is aimed to redefine the `transient_spatial_dependant_porosity_solution_body_force_process`. Now we can control the Reynolds number and a 'Damkohler' setting them as inputs. The viscosity and the permeability are computed from them respectively.

Please mark the PR with appropriate tags: 
- Applications, Features

**Changelog**
Please summarize the changes in one list to generate the changelog:
E.g.
- Added `CalculatePermeability` method.
- Added `CalculateKinematicViscosity` method.

### CODE REVIEW

File: applications/SwimmingDEMApplication/custom_python/add_custom_processes_to_python.cpp [hunk 1]
```diff
@@ -39,7 +39,7 @@ void  AddCustomProcessesToPython(pybind11::module& m)
 
     py::class_<TransientSpatialDependantPorositySolutionBodyForceProcess, TransientSpatialDependantPorositySolutionBodyForceProcess::Pointer, Process>
     (m, "TransientSpatialDependantPorositySolutionBodyForceProcess")
-    .def(py::init< ModelPart&, const double, const double, const double, const double, const double, const double, const double, const double>())
```
Comment (maceligueta) [inline applications/SwimmingDEMApplication/custom_python/add_custom_processes_to_python.cpp:42]: Is it necessary to keep the version with so many arguments? The one below looks more practical, with a Parameters variable, so you have more flexibility in the future, keeping the same API.

Comment (jgonzalezusua) [reply applications/SwimmingDEMApplication/custom_python/add_custom_processes_to_python.cpp:42]: Maybe there is another alternative, but those double are member variables that need to be defined to compute the manufactured solution.

Comment (maceligueta) [reply applications/SwimmingDEMApplication/custom_python/add_custom_processes_to_python.cpp:42]: I was just saying that this constructor looks cooler and the other one, with many arguments, could be considered for deletion.
https://github.com/KratosMultiphysics/Kratos/blob/8736dcd98ed57802a8566f3d9df95c9ea8e3b57a/applications/SwimmingDEMApplication/custom_processes/transient_spatial_dependant_porosity_solution_body_force_process.cpp#L65

Comment (jgonzalezusua) [reply applications/SwimmingDEMApplication/custom_python/add_custom_processes_to_python.cpp:42]: Ok, thanks for the advise. For the moment I will keep in that way until to find out another alternative.

Comment (maceligueta) [reply applications/SwimmingDEMApplication/custom_python/add_custom_processes_to_python.cpp:42]: > I was just saying that this constructor looks cooler and the other one, with many arguments, could be considered for deletion.
> https://github.com/KratosMultiphysics/Kratos/blob/8736dcd98ed57802a8566f3d9df95c9ea8e3b57a/applications/SwimmingDEMApplication/custom_processes/transient_spatial_dependant_porosity_solution_body_force_process.cpp#L65

The other alternative is already coded here, same file, same object. Just informing.

Comment (jgonzalezusua) [reply applications/SwimmingDEMApplication/custom_python/add_custom_processes_to_python.cpp:42]: Ok, I have already modified

**Pull request closed without merging**